### PR TITLE
Refactoring code and directory

### PR DESCRIPTION
**Pull request by UtopiaEH**
## Changelog
```changelog
 - NASA: Description of Change
```
  
## Change Type
- [x] `Added`: new feature - non-breaking change which adds functionality.
- [ ] `Bug fix`: non-breaking change which fixes an issue.
- [ ] `Changed`: for changes in existing functionality.
- [ ] `Deprecated`: for soon-to-be removed features.
- [ ] `Documentation` update.
- [ ] `Performance`: for performance and optimization functionality.
- [ ] `Removed`: for now removed features.
- [ ] `Security`: in case of vulnerabilities.
## Required Checks
 - [ ] Documentation
 - [ ] Unit tests
 - [x] Tested on Chrome
 - [ ] Tested on Firefox
 - [ ] Tested on IE11
 - [x] npm run eslint
 - [x] npm run prettier
## Screenshots
________________

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -18,7 +17,7 @@ const App = () => {
         <div className="container-wrapper">
           <Switch>
             <Route exact path={HOME} component={Header} />
-            <Route path={GALLERY} component={ImageGallery} />
+            <Route path={GALLERY} component={ImagesGalleryScreen} />
```
Comment (cristian-sandu) [inline src/App.js]: why don't you just name it: Gallery?  it is simpler like that:
```
 <Route path={GALLERY} component={Gallery} />
```

File: src/common/components/index.js [hunk 2]
```diff
@@ -1,3 +1,3 @@
-export { default as ImageGallery } from './img-gallery'
+export { default as ImagesGallery } from './Images/Gallery'
```
Comment (cristian-sandu) [inline src/common/components/index.js]: **export { default as Gallery} from './Images/Gallery'**

File: src/common/screens/Footer/index.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { Icon } from 'antd'
 
-import '../scss/Footer.scss'
+import './Footer.scss'
```
Comment (cristian-sandu) [inline src/common/screens/Footer/index.js]: 👍

File: src/App.js (hunk 1 above)
Comment (UtopiaEH) [reply src/App.js]: In the future in the future i would have two components as Images gallery and Video Gallery an it's make the sens

**Pull request closed without merging**